src/utils/storybook/fetchProductHandles.ts

```diff
@@ -0,0 +1,33 @@
+import { getText } from "@/utils/fetch"
+import { createShopifyUrl } from "@/utils/createShopifyUrl"
+
+export async function fetchProductHandles(rootUrl?: string) {
+  if (rootUrl) {
+    window.Shopify = {
+      routes: {
+        root: rootUrl
+      }
+    }
+  }
+
+  const url = createShopifyUrl("collections/all")
+  const response = await getText(url.href)
```

Comment:
the fetch call fails with a CORS error
is it possible to do the fetch all in a way that CORS errors are avoided?